mutualfunds.go

```diff
@@ -77,19 +78,20 @@ type MFSIP struct {
 	DividendType    string `json:"dividend_type"`
 	TransactionType string `json:"transaction_type"`
 
-	Status               string      `json:"status"`
-	SipType              string      `json:"sip_type"`
-	Created              models.Time `json:"created"`
-	Frequency            string      `json:"frequency"`
-	InstalmentAmount     float64     `json:"instalment_amount"`
-	Instalments          int         `json:"instalments"`
-	LastInstalment       models.Time `json:"last_instalment"`
-	PendingInstalments   int         `json:"pending_instalments"`
-	InstalmentDay        int         `json:"instalment_day"`
-	CompletedInstalments int         `json:"completed_instalments"`
-	NextInstalment       string      `json:"next_instalment"`
-	TriggerPrice         float64     `json:"trigger_price"`
-	Tag                  string      `json:"tag"`
+	Status               string           `json:"status"`
+	SipType              string           `json:"sip_type"`
+	Created              models.Time      `json:"created"`
+	Frequency            string           `json:"frequency"`
+	InstalmentAmount     float64          `json:"instalment_amount"`
+	Instalments          int              `json:"instalments"`
+	LastInstalment       models.Time      `json:"last_instalment"`
+	PendingInstalments   int              `json:"pending_instalments"`
+	InstalmentDay        int              `json:"instalment_day"`
+	CompletedInstalments int              `json:"completed_instalments"`
+	NextInstalment       string           `json:"next_instalment"`
+	TriggerPrice         float64          `json:"trigger_price"`
+	StepUp               *json.RawMessage `json:"step_up"`
```

Comment:
Is there a predefined structure for `StepUp`? Instead of `json.RawMessage` we can define it here and use that.

Also might be good to document it here:

https://kite.trade/docs/connect/v3/mutual-funds/#retrieving-an-individual-order

And update the mock and unit test here:

https://github.com/zerodha/kiteconnect-mocks/blob/main/mf_orders.json

Reply:
This is also updated to a struct..